subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -1286,6 +1296,12 @@ static int isr_rx_scan_report(struct lll_scan *lll, uint8_t rssi_ready,
 				ftr->radio_end_us =
 					radio_tmr_end_get() -
 					radio_rx_chain_delay_get(lll->phy, 1);
+
+				ftr->extra =
+					isr_rx_scan_aux_setup(lll, pdu_adv_rx);
+				if (ftr->extra) {
+					err = -EBUSY;
```

Comment:
This error code does not affect caller of the function: `isr_rx_pdu`. It is used as a local flag to check if node has to be enqueued towards ULL. 

```suggestion
					scan_aux_en = true;
```

Reply:
It does affect caller because it will then `return 0` which in turn will affect its called not to disable radio. This is because `isr_rx_scan_aux_setup` already disabled radio to proceed with aux scheduling, disabling radio "as usual" would overwrite that.